apis/v1alpha2/gatewayclass_types.go

```diff
@@ -28,6 +28,7 @@ import (
 // +kubebuilder:storageversion
 // +kubebuilder:printcolumn:name="Controller",type=string,JSONPath=`.spec.controller`
 // +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
+// +kubebuilder:printcolumn:name="Description",type=string,JSONPath=`.spec.description`
```

Comment:
We may want to set a priority field here. That would enable us to [only include this in the wide output](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#priority).

```suggestion
// +kubebuilder:printcolumn:name="Description",type=string,JSONPath=`.spec.description`,priority=1
```